nautobot/docs/plugins/development.md

```diff
@@ -1048,6 +1048,8 @@ class YourAppModelUIViewSet(NautobotUIViewSet):
     ...
 ```
 
++/- 2.0.0
+    The default `lookup_field` for `NautobotUIViewSet` has been changed from `"slug"` to `"pk"`.
 !!! note
```

Comment:
```suggestion
    The default `lookup_field` for `NautobotUIViewSet` has been changed from `"slug"` to `"pk"`.

!!! note
```

Reply:
I think there's a bug in github's handling of suggested changes that include empty lines...